ideascube/mediacenter/migrations/0013_escape_summary.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from django.db import migrations
+from django.utils.html import escape
+
+
+def escape_summary(app, schema_editor):
+    Document = app.get_model('mediacenter', 'Document')
+    db_alias = schema_editor.connection.alias
+    for document in Document.objects.using(db_alias).all():
+        document.summary = "<p>{}</p>".format(escape(document.summary))
```

Comment:
> For example, wrapping a \<div> in a \<p> is invalid HTML, and could render badly.

There will be no `<div>` nor any other tag because we are wrapping plain text.
The text we are wrapping is supposed to be plain text because it was written without tinyMCE and more over, we escape it, so it cannot be something else that plain text.